docs/_includes/layouts/service-patterns.njk

```diff
@@ -3,7 +3,7 @@
 {% block content %}
     <div class="app-layout__content">
         <h1 class="govuk-heading-xl">
-          {% if subsection %}<span class="govuk-caption-xl">{{ subsection }}</span>{% endif %}
+          {% if not isIndex %}<span class="govuk-caption-xl">Service patterns</span>{% endif %}
```

Comment:
I don't think this is working as intended as there's still a caption above the H1 on `/service-patterns/index.md`.

For this to work we'd need to add `isIndex: true` at the top of `/service-patterns/index.md`.  Tested this locally and it works.

Reply:
I think this might be in the templates as a hangover from pre-side nav days when there were index pages for components, patterns, pages.  It does the job in this case though if we do want to use it.

Reply:
Ah ok, cool.  I was thrown as there wasn't a caption above `index.md` in a previous commit.